Clinical-Genomics/scout
# Make variants cytoband genome-build aware

Fix #1908

**How to test**:
1. Modify the config file of one of the 2 test cases so the case genome build is "38"
1. Run `scout setup demo`
1. Make sure that the genome build of variants is saved correctly to database.
1. The best case scenario would be to load a variant that has different cytoband in genome build 37 and 38, but it's complicated because we have quite a few variants in the demo case. I've introduced  a specific test in tests anyway.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by ML
- [x] tests executed by CR, github actions
